packages/orchestration/src/orchestration-client.test.ts

```diff
@@ -341,9 +341,7 @@ describe('orchestration service client', () => {
         config: {
           filters: [
             {
-              id: 'filter1',
               data_repositories: ['*'],
-              search_config: {},
```

Comment:
So why exactly was search_config in there in the first place?
It seems to have been optional even before the change?

Reply:
IDK, but I figured it's setting a bad example and I'll just clean it up 😄